experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -18,13 +18,13 @@ import {
   INVALID_SPAN_CONTEXT,
   propagation,
   ROOT_CONTEXT,
-  Span,
   SpanKind,
   SpanOptions,
   SpanStatus,
   SpanStatusCode,
   trace,
 } from '@opentelemetry/api';
+import { Span } from '@opentelemetry/sdk-trace-base';
```

Comment:
The build is failing because you removed the API span type and added the SDK span type but that isn't allowed. The type returned by API methods will always be the API span type and you can't use SDK methods on it.

Reply:
I reverted this to the initial status because that failed as well.